### PR TITLE
fix-import-issue

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 1]
```diff
@@ -1329,7 +1329,10 @@ private com.google.gwt.event.dom.client.ClickHandler clickHandlerForUploadFromFi
         return event -> {
             // filter only valid items
             if (bucketsListBox.getSelectedIndex() > 0 && workflowsListBox.getSelectedIndex() > 0) {
-                String selectedWorkflowLabel = workflowsListBox.getSelectedValue();
+                String selectedWorkflowLabel = workflowsListBox.getSelectedValue().substring(
```
Comment (mboussaa) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java:1332]: ```workflowsListBox.getSelectedValue().substring(workflowsListBox.getSelectedValue().indexOf(" - ").trim()```

Comment (mboussaa) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java:1332]: and this is wrong if the project name contains one or many ```-```. Same for WF name

**Review by mboussaa — CHANGES_REQUESTED** [review]
to fix

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 1 above)
Comment (mboussaa) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java:1332]: And finally, we should investigate the dropdown list with columns

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 2]
```diff
@@ -1329,7 +1334,13 @@ private com.google.gwt.event.dom.client.ClickHandler clickHandlerForUploadFromFi
...
             // filter only valid items
             if (bucketsListBox.getSelectedIndex() > 0 && workflowsListBox.getSelectedIndex() > 0) {
-                String selectedWorkflowLabel = workflowsListBox.getSelectedValue();
+                String selectedWorkflowLabel = workflowsListBox.getSelectedValue()
+                                                               .contains("\t\t-\t\t") ? workflowsListBox.getSelectedValue()
+                                                                                                        .substring(workflowsListBox.getSelectedValue()
+                                                                                                                                   .indexOf("\t\t-\t\t") +
+                                                                                                                   5)
+                                                                                      : workflowsListBox.getSelectedValue();
+                LogModel.getInstance().logCriticalMessage("selectedWorkflowLabel: " + selectedWorkflowLabel);
```
Comment (mboussaa) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java:1343]: why critical ?

**Review by mboussaa — APPROVED** [review]

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 2 above)
Comment (oanaBiancaSchiopu) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java:1343]: I will delete this

**Review by AlexFalessi — APPROVED** [review]

**Pull request merged**